assets/js/modules/tagmanager/datastore/versions.test.js

```diff
@@ -22,18 +22,10 @@
 import API from 'googlesitekit-api';
 import { STORE_NAME } from './constants';
 import { STORE_NAME as CORE_SITE, AMP_MODE_PRIMARY, AMP_MODE_SECONDARY } from '../../../googlesitekit/datastore/site/constants';
-import {
-	createTestRegistry,
-	muteFetch,
-	untilResolved,
-	unsubscribeFromAll,
-} from '../../../../../tests/js/utils';
+import { createTestRegistry, muteFetch, untilResolved, unsubscribeFromAll } from '../../../../../tests/js/utils';
+import { createBuildAndReceivers, parseLiveContainerVersionIDs as parseIDs } from './__factories__/utils';
```

Comment:
Please keep the one-per-line formatting as it was before.

It looks like the grouping was mostly re-arranged here (moving `__factories__/utils'` imports up) – which is fine.

Reply:
Updated.